fast/stages/0-bootstrap/outputs.tf

```diff
@@ -109,6 +109,7 @@ locals {
         vpcsc       = module.automation-tf-vpcsc-sa.email
         vpcsc-r     = module.automation-tf-vpcsc-r-sa.email
       }
+      default_alerts_email = var.default_alerts_email
```

Comment:
this should go in globals, but TBH I would not even pass this down as it's specific to stage 0